core/src/main/scala/org/http4s/AttributeMap.scala

```diff
@@ -11,22 +11,14 @@ package org.http4s
 /** A key in an [[AttributeMap]] that constrains its associated value to be of type `T`.
   * The key is uniquely defined by its reference: there are no duplicate keys, even
   * those with the same name and type. */
-final class AttributeKey[T] private (val name: String) {
+final class AttributeKey[T] private {
   def apply(value: T): AttributeEntry[T] = AttributeEntry(this, value)
-
-  override def toString: String = name
 }
 
 object AttributeKey {
 
-  /** Construct an [[AttributeKey]] */
-  def apply[T](name: String): AttributeKey[T] = new AttributeKey(name)
-
-  /**
-   * Encourage greater consistency in internal keys by imposing a universal prefix.
-   */
-  private[http4s] def http4s[T](name: String): AttributeKey[T] =
-    apply(s"org.http4s.$name")
+  /** Construct a new [[AttributeKey]] */
+  def apply[T]: AttributeKey[T] = new AttributeKey()
```

Comment:
Would it make sense to leave a deprecated version of the `apply(String)` version to explain why it's ignored and where it went for people who are migrating?